front/src/reducers/osrdconf/types.ts

```diff
@@ -89,6 +89,14 @@ export type PathStep = PathItemLocation & {
     trackNumber: number;
   };
   isInvalid?: boolean;
+  /** Flag specifying whether the pathStep was created from the power restriction selector or not
+   *
+   * If true, the pathStep might be cleaned if its power restriction is removed (except if it has time, stop or margin constraints)
+   *
+   * This flag will only work if the user has not saved its change. Once the change is saved, the flag will be removed and the pathStep
```

Comment:
Nit: the "its" pronoun is not used for words referring to people, such as "the user". "their" can be used instead.